src/redux/actions.js

```diff
@@ -0,0 +1,10 @@
+import * as types from './actionTypes';
+import { isDisplayable } from '../pressButtonLogic';
+
+export default function pressButton(value) {
+    return {
+        type: types.PRESS_BUTTON,
+        value: value,
```

Comment:
You can write here only `value`, no need to specify the key if it matches variable name.